vgstation13.dme

```diff
@@ -1932,6 +1932,7 @@
 #include "maps\defficiency\areas.dm"
 #include "maps\defficiency\pipes.dm"
 #include "maps\randomvaults\objects.dm"
+#include "maps\randomvaults\spacegym.dmm"
```

Comment:
This was merged, it won't update the PR.

I made a seperate one that fixed this right after reading unid's comment.